util/pyclient/test_msgs.py

```diff
@@ -28,20 +28,29 @@ def test_unpack_reply(self):
         self.assertEqual(msg, unpacked_msg)
 
     def test_unpack_request(self):
-        msg = b'hello'
         client_id = 4
         req_seq_num = 1
         read_only = True
         timeout_milli = 5000
+        span_context = b'span context'
+        msg = b'hello'
         cid = str(req_seq_num)
-        packed = bft_msgs.pack_request(client_id, req_seq_num, read_only, timeout_milli, cid, msg)
-        (header, unpacked_msg, cid_) = bft_msgs.unpack_request(packed, len(cid))
-        self.assertEqual(cid, cid_)
+
+        packed = bft_msgs.pack_request(client_id, req_seq_num, read_only,
+                                timeout_milli, cid, msg, False, span_context)
```

Comment:
Can we please avoid boolean parameters? If we must use them, let's make them keyword parameters. With a boolean parameter, it's impossible to tell at the call sight what it means.

Reply:
Actually, I haven't introduced this boolean, but nevertheless I've made it keyword

Reply:
Thanks!